src/solver/model_getters.jl

```diff
@@ -97,6 +97,20 @@ function get_surface_model(parsed_args)
     end
 end
 
+function get_surface_albedo_model(parsed_args, ::Type{FT}) where {FT}
+    albedo_name = parsed_args["variable_surface_albedo"]
+    return if albedo_name in ("false", false, "ConstantAlbedo")
+        ConstantAlbedo{FT}()
+    elseif albedo_name in ("true", true, "RegressionFunctionAlbedo")
+        isnothing(parsed_args["rad"]) && error(
+            "Radiation model not specified, so cannot use RegressionFunctionAlbedo",
+        )
+        RegressionFunctionAlbedo{FT}()
+    else
+        error("Uncaught surface albedo model `$albedo_name`.")
+    end
+end
```

Comment:
I don't think we should support all these options. It becomes messy and unmanagable very quickly. What would true/false be when we add a third option?

I'd suggest calling the option `albedo_model` and sticking with only `ConstantAlbedo` or `RegressionFunctionAlbedo`.

Reply:
Good point! 👍